src/datepicker/datepicker.component.ts

```diff
@@ -16,6 +16,7 @@ import rangePlugin from "flatpickr/dist/plugins/rangePlugin";
 import flatpickr from "flatpickr";
 import { NG_VALUE_ACCESSOR } from "@angular/forms";
 import { carbonFlatpickrMonthSelectPlugin } from "./carbon-flatpickr-month-select";
+import * as languages from "flatpickr/dist/l10n/index";
```

Comment:
Could we export this from the datepicker module, and have the input take the full object instead?

Reply:
When I try to export from the datepicker module I get the error:

Reply:
I get the following error when I export FlatpickrLanguages from the datepicker module:
```
Type 'Record<key, CustomLocale>' is not assignable to type 'any[] | Type<any>'.
  Type 'Record<key, CustomLocale>' is missing the following properties from type 'Type<any>': apply, call, bind, prototype, and 5 more
```

Reply:
`export * as languages from "flatpickr/dist/l10n/index";` or even just leaving the import and doing `export languages;` causes that?